docs/content/operator/ssfn_guide.mdx

```diff
@@ -66,35 +66,16 @@ iota keytool generate ed25519
 
 Create or modify your SSFN's configuration file with these specific settings:
 
-```yaml
-# Network Configuration
-network-key-pair:
-  path: /opt/iota/key-pairs/network.key
-
-# Peering Configuration
-p2p-config:
-  seed-peers:
-    - address: /dns/validator-hostname/udp/8084
-      peer-id: <validator-peer-id>  # Get this from your validator's network.key
-
-# Resource Optimization
-enable-index-processing: false
-
-# Pruning Configuration
-authority-store-pruning-config:
-  num-latest-epoch-dbs-to-retain: 3
-  epoch-db-pruning-period-secs: 3600
-  max-checkpoints-in-batch: 10
-  max-transactions-in-batch: 1000
-  num-epochs-to-retain: 0
-  num-epochs-to-retain-for-checkpoints: 2
-  periodic-compaction-threshold-days: 1
-
-# Metrics Configuration
-metrics:
-  push-interval-seconds: 60
-  push-url: https://metrics-proxy.mainnet.iota.cafe:8443/publish/metrics
-```
+<Tabs groupId="network">
```

Comment:
```suggestion
<Tabs groupId="network" queryString>
```

Allows us to have a query in the url to point people directly to the correct tab/config

https://docusaurus.io/docs/markdown-features/tabs#query-string